openassessment/fileupload/backends/filesystem.py

```diff
@@ -0,0 +1,108 @@
+
+from .base import BaseBackend
+from .. import exceptions
+
+from django.conf import settings
+import django.core.cache
+from django.core.urlresolvers import reverse
+
+
+class Backend(BaseBackend):
```

Comment:
IMHO that would be a bit redundant with the module name: "s3.py" and "filesystem.py". In fileupload/backends/__init__.py, the backends are refered to as "s3.Backend" and "filesystem.Backend".